src/enkf/gridio_fv3reg.f90

```diff
@@ -682,9 +682,6 @@ subroutine writegriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,vargrid
 
   end subroutine writegriddata
   subroutine writeincrement(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,grdin,no_inflate_flag)
- !Dummy subroutine declaration in place of  the actual subroutine definition in
```

Comment:
Why were those comments removed here and in the following dummy subroutines?  Any specific considerations?

Reply:
No specific reason. Just for fixing the conflict with the master GSI. If it is necessary, I can add it back.

Reply:
All the comment lines with "Dummy..." have been put back.